### ISSUE TITLE
MOO Benchmark - JAHS-Bench

### ISSUE THREAD

**Issue by Deathn0t** [deephyper/benchmark#14]
@thchang is currently working on it in https://github.com/thchang/benchmark/tree/main/lib/JAHSBench

!!! NOT COMPATIBLE WITH MACOS !!!

**Comment by Deathn0t** [comment]
Hey @thchang, I can't remember if this was merged in `main` and the issue should be closed?

**Comment by thchang** [comment]
No, the code is working on my fork, but has not been merged into main because i haven't had time to document the usage and update the README files, etc.
